osu.Server.DifficultyCalculator/Program.cs

```diff
@@ -12,7 +12,6 @@ namespace osu.Server.DifficultyCalculator
     [Subcommand(typeof(AllCommand))]
     [Subcommand(typeof(FilesCommand))]
     [Subcommand(typeof(BeatmapsCommand))]
-    [Subcommand(typeof(SinceCommand))]
```

Comment:
not removing the actual command class as well?

Reply:
Either rider is drunk or I am drunk.